docker/docker_client.go

```diff
@@ -1084,3 +1086,9 @@ func (c *dockerClient) getExtensionsSignatures(ctx context.Context, ref dockerRe
 func sigstoreAttachmentTag(d digest.Digest) string {
 	return strings.Replace(d.String(), ":", "-", 1) + ".sig"
 }
+
+// Close removes resources associated with an initialized dockerClient, if any.
```

Comment:
Please update the documentation of `newDockerClient` and `newDockerClientFromRef` to say that the caller must call `.Close()` when done.